index.html

```diff
@@ -6522,15 +6498,7 @@ <h3>Proofs (Signatures)</h3>
 before which the [=credential=] should not be considered [=verified=],
 distinct from the validity period of the credential. This property describes the
 validity of the proof, not of the credential.
-<br/><br/>
-The `verificationMethod` [=property=] specifies, for example, the
-public key that can be used to verify the digital signature. Dereferencing a
-public key URL reveals information about the controller of the key, which can
-be checked against the issuer of the [=credential=]. The
-`proofPurpose` [=property=] clearly expresses the purpose for
-the proof and ensures this information is protected by the signature. A proof is
-typically attached to a [=verifiable presentation=] for authentication
-purposes and to a [=verifiable credential=] as a method of assertion.
+The JWT <code>iat</code> claim likewise provides the time that the signature was made.
```

Comment:
The new line belongs with the preceding (retained) paragraph. The undeleted lines should be a distinct (subsequent) paragraph.
```suggestion
The JWT <code>iat</code> claim likewise provides the time that the signature was made.
<br/><br/>
The `verificationMethod` [=property=] specifies, for example, the
public key that can be used to verify the digital signature. Dereferencing a
public key URL reveals information about the controller of the key, which can
be checked against the issuer of the [=credential=]. The
`proofPurpose` [=property=] clearly expresses the purpose for
the proof and ensures this information is protected by the signature. A proof is
typically attached to a [=verifiable presentation=] for authentication
purposes and to a [=verifiable credential=] as a method of assertion.
```